src/content/community/releases/_index.en.md

```diff
@@ -4,6 +4,26 @@ title = "Releases"
 weight = 40
 +++
 
+## v0.9.1-m1
+
+* `master` branches across all the projects are renamed to `devel`
+* Gateway engine is renamed to `submariner-gateway` from `submariner`
+* Helm charts support submariner deployments using the operator
+* Broker creation is managed by the operator inside of subctl
+* Operator and individual applications have separate service accounts with appropriate privileges
+* For Openshift, Lighthouse CoreDNS metrics are integrated with other CoreDNS metrics
+* KIND deployments support OVS CNI
+* `submariner_connections` is renamed to `submariner_requested_connections` in Prometheus for OpenShift clusters
+* Clusters can be connected with only Lighthouse and no dataplane connectivity via subctl
+* Added support for Flannel in k3s deployments
+* All Submariner metrics are prefixed with `submariner_` in Prometheus for OpenShift clusters
+* Services with Global IPs can now be exported
+* `lighthouse-coredns` no longer relies on `Gateway` object to determine `localClusterID`
```

Comment:
This is an internal detail, ie not user-facing.

Reply:
Someone familiar with inernal details can get confused with these changes are not mentioned somewhere.

Reply:
But the website release notes aren't for devs or folks that are familiar with the internal code - it's for end users. So we try to keep these release notes higher level pertaining to end-user funtionality.